react/global.css

```diff
@@ -1,3 +1,47 @@
-.vtex-impersonate-customer {
+.vtex-telemarketing {
+  text-align: center;
+}
+
+.vtex-telemarketing .vtex-popover__arrow-up {
+  background-color: rgba(0, 0, 0, 0.9);
```

Comment:
use `bg-black-90`

Reply:
Can't use it because i need to override the class